src/app/public/theming/theme-if.directive.ts

```diff
@@ -72,11 +76,14 @@ export class SkyThemeIfDirective implements OnDestroy {
   private updateView(): void {
     const condition = this.context && this.currentTheme?.theme.name === this.context;
     if (condition && !this.hasView) {
-      this.viewContainer.createEmbeddedView(this.templateRef);
       this.hasView = true;
+      this.embeddedView = this.viewContainer.createEmbeddedView(this.templateRef);
```

Comment:
Do we still need the `EmbeddedViewRef`?

Reply:
I'm using the ref to call `this.embeddedView.destroy();` when the condition flips.